resources/views/livewire/administrative-tools/create-event.blade.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use App\Models\Achievement;
+use App\Models\EventAchievement;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use App\Platform\Actions\ComputeGameSortTitleAction;
+use App\Platform\Enums\AchievementFlag;
+use Filament\Forms;
+use Filament\Forms\Concerns\InteractsWithForms;
+use Filament\Forms\Contracts\HasForms;
+use Filament\Forms\Form;
+use Filament\Notifications\Notification;
+use Illuminate\Support\Facades\Auth;
+use Livewire\Volt\Component;
+
+new class extends Component implements HasForms {
+    use InteractsWithForms;
+
+    public ?string $title = null;
+    public int $numberOfAchievements = 6;
+
+    public function submit(): void
+    {
+        // Validate.
+        $this->form->getState();
+
+        $eventTitle = $this->title ?? 'New Event';
+        $event = Game::Create([
+            'Title' => $eventTitle,
+            'sort_title' => (new ComputeGameSortTitleAction())->execute($eventTitle),
+            'Publisher' => 'RetroAchievements',
+            'ConsoleID' => System::Events,
+        ]);
+
+        for ($i = 0; $i < $this->numberOfAchievements; $i++) {
+            $achievement = Achievement::Create([
```

Comment:
I didn't know PHP function names are case-insensitive. That's probably why this still works. I think we should change `Create()` to `create()` though to stick with convention.